UI/window-basic-auto-config.cpp

```diff
@@ -1000,6 +1000,17 @@ void AutoConfig::TestHardwareEncoding()
 			hardwareEncodingAvailable = qsvAvailable = true;
 		else if (strcmp(id, "h264_texture_amf") == 0)
 			hardwareEncodingAvailable = vceAvailable = true;
+#ifdef __APPLE__
+		else if (strcmp(id,
+				"com.apple.videotoolbox.videoencoder.ave.avc")
```

Comment:
I think you're missing a `== 0` here

Reply:
You're definitely right. I'm honestly a bit baffled why it still worked.

Reply:
fixed